2022-02-02 15:49:36,122 INFO    MainThread:17564 [wandb_setup.py:_flush():71] setting env: {}
2022-02-02 15:49:36,123 INFO    MainThread:17564 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-02 15:49:36,123 INFO    MainThread:17564 [wandb_init.py:_log_setup():371] Logging user logs to H:\PROJECT\PYTHON\FGVSR\wandb\run-20220202_154936-3z32gfh3\logs\debug.log
2022-02-02 15:49:36,123 INFO    MainThread:17564 [wandb_init.py:_log_setup():372] Logging internal logs to H:\PROJECT\PYTHON\FGVSR\wandb\run-20220202_154936-3z32gfh3\logs\debug-internal.log
2022-02-02 15:49:36,123 INFO    MainThread:17564 [wandb_init.py:init():404] calling init triggers
2022-02-02 15:49:36,123 INFO    MainThread:17564 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-02 15:49:36,123 INFO    MainThread:17564 [wandb_init.py:init():449] starting backend
2022-02-02 15:49:36,123 INFO    MainThread:17564 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-02-02 15:49:36,127 INFO    MainThread:17564 [backend.py:ensure_launched():199] starting backend process...
2022-02-02 15:49:36,406 INFO    MainThread:17564 [backend.py:ensure_launched():204] started backend process with pid: 23180
2022-02-02 15:49:36,406 INFO    MainThread:17564 [wandb_init.py:init():458] backend started and connected
2022-02-02 15:49:36,407 INFO    MainThread:17564 [wandb_init.py:init():519] updated telemetry
2022-02-02 15:49:36,408 INFO    MainThread:17564 [wandb_init.py:init():549] communicating current version
2022-02-02 15:49:37,514 INFO    MainThread:17564 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-02 15:49:37,515 INFO    MainThread:17564 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-02 15:49:37,976 INFO    MainThread:17564 [wandb_init.py:init():592] starting run threads in backend
2022-02-02 15:49:40,539 INFO    MainThread:17564 [wandb_run.py:_console_start():1816] atexit reg
2022-02-02 15:49:40,539 INFO    MainThread:17564 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-02-02 15:49:40,539 INFO    MainThread:17564 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-02-02 15:49:40,541 INFO    MainThread:17564 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-02 15:49:40,541 INFO    MainThread:17564 [wandb_init.py:init():619] run started, returning control to user process
2022-02-02 15:49:40,582 INFO    MainThread:17564 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-02-02 15:49:40,582 INFO    MainThread:17564 [wandb_run.py:_restore():1758] restore
2022-02-02 15:49:41,559 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 548
}

2022-02-02 15:49:42,019 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 548
}

2022-02-02 15:49:43,023 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 10124
}

2022-02-02 15:49:43,132 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 10150
}

2022-02-02 15:49:43,238 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 10150
}

2022-02-02 15:49:43,343 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 10150
}

2022-02-02 15:49:43,448 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 10150
}

2022-02-02 15:49:43,554 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 548
  total_bytes: 10150
}

2022-02-02 15:49:43,658 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 574
  total_bytes: 10150
}

2022-02-02 15:49:43,763 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 574
  total_bytes: 10150
}

2022-02-02 15:49:43,868 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2976
  total_bytes: 10150
}

2022-02-02 15:49:43,973 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2976
  total_bytes: 10150
}

2022-02-02 15:49:44,078 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,183 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,288 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,394 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,499 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,604 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,709 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:44,814 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:45,074 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}

2022-02-02 15:49:46,100 INFO    MainThread:17564 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10150
  total_bytes: 10150
}
local_info {
}

2022-02-02 15:49:47,207 INFO    MainThread:17564 [wandb_run.py:_append_files():2184] logging synced files
